experiments/ClimaEarth/run_cloudless_aquaplanet.jl

```diff
@@ -0,0 +1,362 @@
+# # Cloudless Aquaplanet
```

Comment:
Could you add a couple sentences at the top here explaining the setup for this simulation, and particularly how it's different from the other drivers? It would be good actually to have this for each of the driver scripts, especially as we're adding more

Reply:
Of course, good point!